### PR TITLE
ligolo-ng: init at 0.1

### PR DESCRIPTION
**Pull request by elohmeier**
Added package useful for pentesting. #81418

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/networking/ligolo-ng/default.nix [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+  vendorSha256 = "sha256-axRCThmFavR+GTRWSgdAr2mbrp07hsFea0rKLQNIhgU=";
+
+  doCheck = false; # tests require network access
+
+  meta = with lib; {
+    homepage = "https://github.com/tnpitsecurity/ligolo-ng";
+    description = "A tunneling/pivoting tool that uses a TUN interface";
+    platforms = platforms.linux;
+    license = licenses.gpl3;
```
Comment (fabaff) [inline pkgs/tools/networking/ligolo-ng/default.nix:28]: License looks like GPLv3. There are no statements that later version are supported.

```suggestion
    license = licenses.gpl3Only;
```

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 138469` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>ligolo-ng</li>
  </ul>
</details>

File: pkgs/tools/networking/ligolo-ng/default.nix (hunk 1 above)
Comment (elohmeier) [reply pkgs/tools/networking/ligolo-ng/default.nix]: Thanks, fixed.

**Review by fabaff — APPROVED** [review]
Looks good to me.

**Pull request merged**